data/users.csv

```diff
@@ -0,0 +1,6 @@
+UserId,Name,Email,Cellphone
+1,Ana Garcia Valverde,ana@hotmail.com,33 33 33 33 33
+2,Blanca Garcia Valverde,blanca@hotmail.com,33 33 33 33 33
+3,Claudia Garcia Valverde,claudia@hotmail.com,33 33 33 33 33
+4,Daniela Garcia Valverde,daniela@hotmail.com,33 33 33 33 33
+5,Erika Garcia Valverde,erika@hotmail.com,33 33 33 33 33
```

Comment:
The CSV file should have an empty line at the end because in the case your app adds another registry, it will be added in the same row, so you will have two registries in the same line.